pkgs/applications/window-managers/notion/system-local-nixos.mk

```diff
@@ -0,0 +1,11 @@
+# NixOS config file
+CC=cc
+
+# PRELOAD_MODULES needs to be 0;
+# a bug prevents Notion to be compiled
+# if this variable is set to 1
+# https://github.com/raboof/notion/issues/202
+PRELOAD_MODULES=0
+
+# Purity reasons
+X11_PREFIX="/no-such-path"
```

Comment:
Can these all not just be set buildFlags?

Reply:
Done